apps/include/apps.h

```diff
@@ -263,9 +263,9 @@ extern char *psk_key;
 
 unsigned char *next_protos_parse(size_t *outlen, const char *in);
 
-void print_cert_checks(BIO *bio, X509 *x,
+int check_cert(BIO *bio, X509 *x,
```

Comment:
I still do not like the name of the function as this one does not really describe what the function does. Perhaps `check_cert_data` or something like that?

Reply:
As these data are the certificate attributes, I named the function as `check_cert_attributes`